### ISSUE TITLE
Support animation type

### ISSUE THREAD

**Issue by Nikoloutsos** [BastiaanJansen/toast-swift#24]
Thinking of making a PR that will give the ability to choose the toast entering/exiting animation.

```diff
let config = ToastConfiguration(
    direction: .up,
    autoHide: true,
    enablePanToClose: true,
    displayTime: 5,
    animationTime: 0.2
+   animationType: .slide 
)
```

also `fade` animation type can be an option.

What do you think 🤔

**Comment by BastiaanJansen** [comment]
That would be cool. Although I do think the user should also be able to provide their own animation, but can also choose between a couple of already built-in animations.

I say go for it!

**Comment by BastiaanJansen** [comment]
Thanks @Nikoloutsos! Released in version 1.3.0.
